public/app/features/explore/NodeGraph/NodeGraphContainer.tsx

```diff
@@ -81,12 +84,13 @@ export function UnconnectedNodeGraphContainer(props: Props) {
     ) : null;
 
   return (
-    <Collapse
-      label={<span>Node graph{countWarning} </span>}
-      collapsible={withTraceView}
+    <PanelChrome
+      title={`Node graph`}
+      titleItems={countWarning}
       // We allow collapsing this only when it is shown together with trace view.
-      isOpen={withTraceView ? open : true}
-      onToggle={withTraceView ? () => toggled() : undefined}
+      collapsible={!!withTraceView}
+      collapsed={withTraceView ? collapsed : false}
```

Comment:
Do you need to control the state of the component? Can you leave the component uncontrolled?

Reply:
I do need to control it because when trace view is enabled, the component needs to be collapsed by default, while uncontrolled one would be expanded by default.